browser-extension/manifest.json

```diff
@@ -2,10 +2,21 @@
   "manifest_version": 2,
 
   "name": "Starfish",
-  "version": "2",
-  "description": "Open in IDE",
+  "version": "1.0",
+  "description": "Utility  Extension for Starfish.It Intends to  add an \"Open in IDE\" button on Github and Gitlab.Easily Configurable to chose between cloning Protocols viz HTTPS and SSH",
```

Comment:
there is no gitlab functionallity thus far, right ? lets not that in description before it is there.

Reply:
sure max 👍